python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1734,6 +1779,19 @@ def create_reexecuted_run(
                 parent_run=parent_run,
             )
             tags[RESUME_RETRY_TAG] = "true"
+        elif strategy == ReexecutionStrategy.FROM_ASSET_FAILURE:
+            skipped_asset_keys, skipped_asset_check_keys = self._get_skipped_entity_keys(
+                parent_run_id
+            )
+            remote_job = code_location.get_job(
+                remote_job.get_subset_selector(
+                    asset_selection=skipped_asset_keys,
+                    asset_check_selection=skipped_asset_check_keys,
+                )
+            )
+            step_keys_to_execute = None
+            known_state = None
+            tags[RESUME_RETRY_TAG] = "true"
```

Comment:
not sure if we want to set the exact same tag in this slightly different behavior path

Reply:
yeah looking at this more closely it seems wise to distinguish between these cases